acceptance-tests/src/test/java/uk/gov/di/test/utils/RetryHelper.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.di.test.utils;
+
+import java.util.HashMap;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class RetryHelper {
+
+    private static volatile RetryHelper instance;
```

Comment:
think volatile is for system or low level development

Reply:
> Every thread that accesses a volatile variable will read it from main memory, and not from the CPU cache. This way, all threads see the same value for the volatile variable.

Reply:
ok nice, genuinely have never seen anyone use this :)